components/dashboard/src/components/delete-account-view.tsx

```diff
@@ -34,87 +34,83 @@ export class DeleteAccountView extends React.Component<DeleteAccountViewProps, D
+                        <CircularProgress />
+                    </DialogContent>
+                </React.Fragment>}
 
-            {this.state.status == "done" && <React.Fragment>
-            <DialogTitle>We are sorry to see you go.</DialogTitle>
-            <DialogContent>
-                <Typography variant="body1">
-                    <div>Your account has been deleted &mdash; all that's left is to log out using the finish button below.</div>
-                    <div>If you ever decide to give Gitpod another try, just sign up like you did the first time. No hard feelings 🙂</div>
-                </Typography>
-            </DialogContent>
-            <DialogActions>
-                <Button onClick={() => window.location.href=new GitpodHostUrl(window.location.toString()).withApi({pathname: '/logout'}).toString()} color="secondary" variant="outlined">Finish</Button>
-            </DialogActions>
-            </React.Fragment>}
+                {this.state.status === "done" && <React.Fragment>
+                    <DialogTitle>We are sorry to see you go.</DialogTitle>
+                    <DialogContent>
+                        <Typography variant="body1">
+                            <div>Your account has been deleted and your browser will be redirected to the logout.</div>
+                            <div>Feel free to try out Gitpod again. No hard feelings 🙂</div>
+                        </Typography>
+                    </DialogContent>
+                    <DialogActions>
+                        <Button onClick={() => this.doLogout()} color="secondary" variant="outlined">Goodbye!</Button>
+                    </DialogActions>
+                </React.Fragment>}
 
-            {this.state.status == "error" && <React.Fragment>
-            <DialogTitle>Something went wrong.</DialogTitle>
-            <DialogContent>
-                <Typography variant="body1">
-                    <div>We are sorry - something went wrong while deleting your account. </div>
-                    <div>Please get in touch at <a href="mailto:support@gitpod.io">support@gitpod.io</a>.</div>
-                </Typography>
-            </DialogContent>
-            <DialogActions>
-                <Button onClick={() => this.setState({ status: undefined })} color="secondary" variant="outlined">Ok</Button>
-            </DialogActions>
-            </React.Fragment>}
-        </Dialog>
+                {this.state.status === "error" && <React.Fragment>
+                    <DialogTitle>Something went wrong.</DialogTitle>
+                    <DialogContent>
+                        <Typography variant="body1">
+                            <div>We are sorry – something went wrong while deleting your account.</div>
+                            <div>Please get in touch at <a href="mailto:support@gitpod.io">support@gitpod.io</a>.</div>
+                            <div>As a precaution, your browser will still be redirected to the logout.</div>
```

Comment:
```suggestion
                            <div>As a precaution you will still be logged out of Gitpod - you can log in any time.</div>
```

Reply:
> you can log in any time.

why? if the account was delete half way, AND the intention was to delete it, then it's not helpful to recommend to log in.

Reply:
Good point.